src/index.ts

```diff
@@ -146,12 +146,92 @@ export class Helius {
             return data;
         } catch (err: any | AxiosError) {
             if (axios.isAxiosError(err)) {
-                throw new Error(`error during appendAddressesToWebhook: ${err.response?.data.error}`)
+                throw new Error(`error during appendAddressesToWebhook: ${err.response?.data.error || err}`)
             } else {
                 throw new Error(`error during appendAddressesToWebhook: ${err}`)
             }
         }
     }
 
+    async createCollectionWebhook(request: CreateCollectionWebhookRequest): Promise<Webhook> {
+        if (request?.collectionQuery == undefined) {
+            throw new Error(`must provide collectionQuery object.`)
+        }
+
+        const { firstVerifiedCreators, verifiedCollectionAddresses } = request.collectionQuery;
+        if (firstVerifiedCreators != undefined && verifiedCollectionAddresses != undefined) {
+            throw new Error(`cannot provide both firstVerifiedCreators and verifiedCollectionAddresses. Please only provide one.`)
+        }
+
+        let mintlist: MintlistObject[] = [];
+        let query = {};
+
+        if (firstVerifiedCreators != undefined) {
+            query = { firstVerifiedCreators }
+        } else { // must have used verifiedCollectionAddresses
```

Comment:
prob worth checking this anyways so we can return a sensical error if somehow happens